gremlin-go/driver/driverRemoteConnection.go

```diff
@@ -19,17 +19,51 @@ under the License.
 
 package gremlingo
 
-import "golang.org/x/text/language"
+import (
+	"crypto/tls"
+	"golang.org/x/text/language"
+	"net/http"
+)
+
+// AuthInfo is an option struct that allows authentication information to be specified.
+// Authentication can be provided via http.Header Header directly.
+// Basic authentication can also be used via the BasicAuthInfo function.
+type AuthInfo struct {
```

Comment:
Since this is also used by the Client, I want to say that this can live in its own file.